src/main/java/traincraft/blocks/assemblytables/AssemblyCraftingItemHandler.java

```diff
@@ -0,0 +1,71 @@
+/*
```

Comment:
The recipe belongs into the tile entity itself.
This class isn't needed, since there is already a inventory class for content change listeners in the api package path. The distillery uses this inventory

Reply:
I moved the recipe into the tileentity, but I cannot find the inventory class you are talking about. The ones that are already in use do too much for the assembly table, or are not ItemHandlers.

Reply:
was able to phase this out when moving to a single inventory